EXTENSION.md

```diff
@@ -145,6 +145,14 @@ Two code actions will appear against a function with this error: _Add public vis
 
 ![Public Private](https://raw.githubusercontent.com/NomicFoundation/hardhat-vscode/main/docs/gifs/public-private.gif "Public Private")
 
+#### Adding license identifier and `pragma solidity` version
+
+When no license is specified on a contract, the `solidity(1878)` warning is raised by the compiler. Similarly, when no compiler version is specified with a `pragma solidity` statement, the compiler shows the `solidity()` warning. There are code actions available for quick fixes.
```

Comment:
compiler shows the `solidity(3420)` warning

Reply:
Nice catch! I was sleepy when writing this one :joy: